accelerator/BUILD

```diff
@@ -34,7 +34,7 @@ cc_binary(
         "-DENABLE_MQTT",
     ],
     deps = [
-        ":ta_config",
+        ":ta_config_mqtt",
```

Comment:
I don't really get the reason why you did such change.

Reply:
The predefine `-DENABLE_MQTT` doesn't work in the bazel rule `ta_config`, so I use the other rule `ta_config_mqtt` which has the predefine `-DENABLE_MQTT` to make MQTT work fine.

Reply:
It is misleading! Can you change the internal configurations?

Reply:
It seems hard to do that. Accroding to the bazel spec https://docs.bazel.build/versions/master/be/c-cpp.html#cc_library.copts
`The flags take effect only for compiling this target, not its dependencies`
So I have to add `copts = [-DENABLE_MQTT]` into all the bazel rules which need this predefinition.

Reply:
Always ensure the identifiers (such as `ta_config`) lean and mean for Bazel.

Reply:
Can we build both `ta_config` and `ta_config_mqtt`?

Reply:
It's OK to have both both target `ta_config` and `ta_config_mqtt` at the same bazel rule, but `ta_config_mqtt` is just `ta_config` with predifinition `ENABLE_MQTT`.

Reply:
We can provide both targets. However, `ta_config_mqtt` shall be optional.

Reply:
But if `accelerator_mqtt` doen't use `ta_config_mqtt`, then it can't work.
Or the optional you mentioned is in other way?

Reply:
It is sad. I don't want MQTT be a build dependency nor runtime library.

Reply:
If I integrate the two bazel rules, `accelerator` and `config`, into one single bazel rule, then it can be optional.
In this situation, we can assign predefined value `ENABLE_MQTT` to build MQTT relates files

Reply:
So, what is the further action?

Reply:
I build a rule, `accelerator_mqtt`, especially for building accelerator with MQTT interface. Is it better than the old way?